contracts/tutorial/src/msg.rs

```diff
@@ -5,7 +5,7 @@ use serde::{Deserialize, Serialize};
 use crate::state::State;
 
 /// A message sent to initialize the contract state.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
```

Comment:
Same comments as the ones for attrs/src/msg.rs, including the ones for QueryMsg.

Reply:
now using `cw_serde`